### ISSUE TITLE
Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".

### ISSUE THREAD

**Issue by davidfowl** [OmniSharp/omnisharp-atom#178]
[Enter steps to reproduce below:]

1. Make an empty web application

**Atom Version**: 0.194.0
**System**: Mac OS X 10.10.3
**Thrown From**: [omnisharp-atom](https://github.com/OmniSharp/omnisharp-atom) package, v0.4.6

### Stack Trace

Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".

```
At events.js:141

EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".

  at Function (native)
  at Function.template (/Users/dfowler/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/cheerio/node_modules/lodash/dist/lodash.js:6305:22)
  at underscore (/Users/dfowler/.atom/packages/omnisharp-atom/node_modules/generator-aspnet/node_modules/yeoman-generator/lib/util/engines.js:23:14)
  at [object Object].actions.engine (/Users/dfowler/.atom/packages/omnisharp-atom/node_modules/generator-aspnet/node_modules/yeoman-generator/lib/actions/actions.js:216:32)
  at [object Object].template (/Users/dfowler/.atom/packages/omnisharp-atom/node_modules/generator-aspnet/node_modules/yeoman-generator/lib/actions/actions.js:191:19)
  at [object Object].yeoman.generators.Base.extend.writing (/Users/dfowler/.atom/packages/omnisharp-atom/node_modules/generator-aspnet/app/index.js:146:18)
  at /Users/dfowler/.atom/packages/omnisharp-atom/node_modules/generator-aspnet/node_modules/yeoman-generator/lib/base.js:409:16
  at processImmediate [as _immediateCallback] (timers.js:369:17)

```

### Commands

```
     -4:22.6.0 editor:newline (atom-text-editor.editor)
     -4:18.6.0 editor:select-to-first-character-of-line (atom-text-editor.editor)
  2x -4:18.1.0 core:backspace (atom-text-editor.editor)
     -4:17.6.0 core:save (atom-text-editor.editor)
     -3:44.1.0 core:move-up (atom-text-editor.editor)
     -3:43.8.0 editor:move-to-end-of-screen-line (atom-text-editor.editor)
     -3:43.6.0 editor:newline (atom-text-editor.editor)
  2x -3:41.3.0 editor:select-to-first-character-of-line (atom-text-editor.editor)
  2x -3:40.9.0 core:backspace (atom-text-editor.editor)
     -3:40.5.0 core:save (atom-text-editor.editor)
     -0:22.3.0 omnisharp-atom:new-application (atom-text-editor.editor)
     -0:15.9.0 editor:consolidate-selections (atom-text-editor.editor.mini)
     -0:15.9.0 core:cancel (atom-text-editor.editor.mini)
     -0:07.6.0 omnisharp-atom:new-application (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-atom-dark-syntax.theme-atom-dark-ui)
  2x -0:06.4.0 core:move-down (atom-text-editor.editor.mini)
  2x -0:05.0 core:confirm (atom-text-editor.editor.mini)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-yeoman, v0.1.5
autocomplete-plus, v2.12.0
language-csharp, v0.5.0
linter, v0.12.1
omnisharp-atom, v0.4.6

# Dev
No dev packages
```

**Comment by david-driscoll** [comment]
This is probably an issue for atom-yeoman

**Comment by david-driscoll** [comment]
this is due to a new change in the latest aspnet generator... and it's nasty...

**Comment by david-driscoll** [comment]
fixed in atom-yeoman 0.2.0, we should bump omnisharp-atoms package.json.

See [the dirty dirty hack here](https://github.com/david-driscoll/atom-yeoman/commit/b325f7ea0f14340a2391d0cdbd3f18fbc88d77ec)

**Comment by david-driscoll** [comment]
Closing as this should be fixed in the latest atom-yeoman.
